orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesProcessor.java

```diff
@@ -0,0 +1,44 @@
+package com.netflix.spinnaker.orca.clouddriver.tasks.servergroup;
+
+import com.netflix.spinnaker.orca.front50.model.Application;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class TitusInterestingHealthProviderNamesProcessor implements InterestingHealthProviderNamesProcessor {
+  private static final String TITUS = "Titus";
+  private static final String INTERESTING_HEALTH_PROVIDER_NAMES = "interestingHealthProviderNames";
+
+  @Override
+  public boolean supports(String cloudProvider, Application application, Stage stage) {
+    if (cloudProvider.equalsIgnoreCase(TITUS)) {
+      if (application.platformHealthOnly == Boolean.TRUE && application.platformHealthOnlyShowOverride == Boolean.TRUE) {
+        return true;
+      }
+
+      // checks titus job label
+      Map<String, Object> labels = (Map<String, Object>)stage.getContext().get("labels");
+      return (labels != null && labels.containsKey(INTERESTING_HEALTH_PROVIDER_NAMES) && String.valueOf(labels.get(INTERESTING_HEALTH_PROVIDER_NAMES)).equalsIgnoreCase(TITUS));
```

Comment:
are we mapping a list to a string in the label? 

It feels like we want to set a more explicit discoveryEnabled label and convert this into interestingHealthProviderNames, but can be convinced otherwise. This doesn't feel very future proof once Titus starts adding ALB support and we can then have [ "Titus", "Discovery" ] or [ "Titus", "Loadbalancer" ]